v3ElectronicsUserGuide.md

```diff
@@ -1601,6 +1602,16 @@ amc_info_uhal.py --shelf=X -sY
 
 This will print various info about the board, the DAQ link status, the TTC status, and the SCA status. Note that `ipbus` must be running on the CTP7.
 
+## Reading a Register Repeatedly
+
+If you would like to repeatedly read the same register (e.g. in order to determine the rate of bits being flipped) execute:
+
+```
+repeated_reg_read.py REGISTER_NAME X Y --card eagleXX
+```
+
+This will read register REGISTER_NAME X times, pausing Y microseconds between each read. Results are written to terminal and also an output text file: [filename].txt Y should be set to > 250 microseconds.
```

Comment:
Seems like you're missing the markdown formatting I had specified [here](https://github.com/AndrewLevin/sw_utils/pull/1#discussion_r232707481)

Reply:
@bdorney, I have added the "`"s now.